tools/awssdkpatch/patch.tmpl

```diff
@@ -59,18 +59,22 @@ var x identifier
 
 # Step 2: Convert AWS SDK v1 (service) to v2
 @@
-var x identifier
+var i, x identifier
```

Comment:
From the best practices section, it sounds like this metavariable should match the import name exactly:
https://github.com/uber-go/gopatch/blob/main/docs/PatchesInDepth.md#best-practices-for-imports

```suggestion
var {{ .ImportAlias }}, x identifier
```

Reply:
So, I originally had the `{{ .ImportAlias }}` variant and thought I'd be clever by making it more generic. I hadn't seen that far down in the docs though; will change it back.